plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -461,54 +536,54 @@ lookupVarName varName varEnv = do
 enterComputeCek
     :: forall uni fun s
     . (Ix fun, PrettyUni uni fun, GivenCekReqs uni fun s, uni `Everywhere` ExMemoryUsage)
-    => CekMachineCosts
-    -> Context uni fun
+    => Context uni fun
     -> CekValEnv uni fun
     -> Term Name uni fun ()
     -> CekM s (Term Name uni fun ())
-enterComputeCek costs = computeCek where
+enterComputeCek = computeCek (toWordArray 0) where
```

Comment:
Provide an everywhere-zero array in the library?

Reply:
I think it makes sense to just expose the function that lets you initialize it at a word: you might want to initialize it at some other literal value, 0 is mostly only special for us since we're using it as a counter.